### DISCUSSION TITLE
External Logstash pipeline for filebeat & winlogbeat event

### DISCUSSION THREAD

**Discussion by n0th1nt0lOOz** [Security-Onion-Solutions/securityonion#5574] [Unsupported Versions]
As we can use external logstash as log source so I want to ask for the suitable pipeline that help the manager node could use

My current logtash pipeline
```
input {
        beats {
                port => 5044
                host => "0.0.0.0"
        }

        tcp {
                port => 5000
        }
}

## Add your filters / logstash plugins configuration here

output {
        elasticsearch {
                hosts => "elasticsearch:9200" // this is my log centralize system
                user => "elastic"
                password => "changeme"
                ecs_compatibility => disabled
                index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}"
        }

        elasticsearch {
                hosts => "192.168.87.47:9200" // Heavy node
                user => "so_elastic"
                password => "nFHrqKrQZV0aRSJYKjf4"
                ecs_compatibility => disabled
                index => "so-beats"
        }
        stdout { codec => rubydebug }
}
```

What should I change? I need winlogbeat and sysmon. And linux log with filebeat in the future.
And I so-allow beats-logstash 5044 from manager node. Is it enought?
Thanks & Regards

**Comment by weslambert** [comment]
I'm not sure I understand.  Are you trying to ship logs from Winlogbeat/Sysmon from another Logstash instance or directly from the agent to Security Onion?

**Reply by n0th1nt0lOOz** [reply]
Hi @weslambert ,
I want to ship logs from another logstash to a heavynode of SO. Do you have any template for that config? Really need that to feed logs for both S.O and IT Log System too
Thanks

**Comment by n0th1nt0lOOz** [comment]
Hi all
I till need help here, please
